Modules/Options/Skins/GameMenuSkin.lua

```diff
@@ -154,6 +157,24 @@ function O:Skins_ElvUI()
       disabled = optionsDisabled,
       hidden = not TXUI.IsRetail,
     }
+
+    infoGroup.historyLimit = {
+      order = self:GetOrder(),
+      type = "range",
+      name = "History Limit",
+      desc = "Number of Mythic+ dungeons shown in the history list.",
+      min = 1,
+      max = 10,
+      step = 1,
```

Comment:
How would the user disable displaying the history? It either needs a separate toggle or allow it to go to 0 and update code to reflect that. From an UX standpoint, separate toggle would make more sense.